iot-e2e-tests/android/app/build.gradle

```diff
@@ -82,10 +82,12 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'com.android.support:multidex:1.0.3'
 
-    implementation ('com.microsoft.azure.sdk.iot:iot-e2e-common:0.26.0'){
-        exclude module: 'junit'
-        exclude module: 'azure-storage'
```

Comment:
In the pom file for the common module, I have excluded these jars from the shade jar, so there is no need to exclude them here anymore